modnet/preprocessing.py

```diff
@@ -687,7 +693,13 @@ def featurize(
 
             global DATABASE
             if DATABASE.empty:
-                DATABASE = pd.read_pickle(db_file)
+                from modnet.ext_data import load_ext_dataset
+
+                db_path = load_ext_dataset("MP_210321", "feature_db")
+                try:
+                    DATABASE = pd.read_pickle(db_path)
+                except AttributeError:
+                    raise AttributeError("Please update pandas to >=1.3")
```

Comment:
We currently have pandas>0.25,<1.2 in our setup.py (I think this was due to incompatibility with some other package) - what is the issue here? Could we just say pandas>1.3?

Reply:
If I remember correctly, I saved the df with pandas 1.3, which makes reading below 1.3 not possible